README.md

```
@@ -2,882 +2,22 @@
+# Espressif-IDE (IDF Eclipse Plugin)
 
-ESP-IDF Eclipse Plugin brings developers an easy-to-use Eclipse-based development environment for developing ESP32-based IoT applications.
-It provides better tooling capabilities, which simplifies and enhances standard Eclipse CDT for developing and debugging ESP32 IoT applications. It offers advanced editing, compiling, flashing and debugging features with the addition of Installing the tools, SDK configuration and CMake editors.
+Espressif-IDE is an Integrated Development Environment (IDE) based on Eclipse CDT for developing IoT Applications using the ESP-IDF. It’s a standalone and customized IDE built specifically for ESP-IDF. Espressif-IDE comes with the IDF Eclipse plugins, essential Eclipse CDT plugins, and other third-party plugins from the Eclipse platform to support building ESP-IDF applications.
 
 The plug-in runs on `macOS`, `Windows` and `Linux` platforms.
 
-![](docs_readme/images/macos-logo.png)
-![](docs_readme/images/windows-logo.png)
-![](docs_readme/images/linux-logo.png)
+![](media/macos-logo.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)